src/guides/reinstall-os.html

```diff
@@ -7,53 +7,53 @@
   <p>
     <span
       >Note: It is very unlikely that reinstalling the Home Assistant OS is necessary. Before going through
-      this procedure, make sure to use the built-in device wipe mechanism.</span
+      this procedure, make sure to use the built-in <a href="/faq/#can-i-factory-reset-the-system">device wipe mechanism</a>.</span
     >
   </p>
-  <p><span></span></p>
   <p>
-    <span
-      >By default, the Raspberry Pi Compute Module 4 has a fixed boot order: it
-      tries to boot from the internal eMMC before booting from a USB mass
-      storage device. In rare cases, the internal eMMC is deemed
-      bootable, but does not boot into a working system. In this case, there are
-      two options you can try to reinstall Home Assistant OS.</span
-    >
+    <span>
+      To reinstall Home Assistant OS the <b>Home Assistant OS Installer for
+      Yellow</b> flashed on a USB flash drive needs to be used. However, the
+      Raspberry Pi Compute Module 4 has a fixed boot order: <b>eMMC</b> -> <b>NVMe SSD</b> -> <b>USB flash drive</b>.
+    </span>
+  </p>
+  <p>
+    <span>
+      Option 1 disables booting of Home Assistant OS (from eMMC or NVMe SSD)
+      which makes it possible to start the Installer again from a USB flash drive.
+      This option only works if you have currently Home Assistant OS installed!
+    </span>
   </p>
-  
-  <p><span></span></p>
   <p>
     <span><h2>Option 1 (recommended): Reinstall Home Assistant OS using Red + Blue button</h2></span>
   </p>
-  <p><span>This option works with Home Assistant OS 8.5 and later only.</span></p>
   <ol start="1">
     <li>
-      <span>Remove power from your system.</span>
+      <span>Remove power from your system. Make sure no USB device is plugged into the device.</span>
```

Comment:
```suggestion
      <span>Make sure no USB device is plugged into the device. Remove power from your system. </span>
```
I would put that first, just in case people pull the plug before reading on..